### DISCUSSION TITLE
I need help on zksync compiling

### DISCUSSION THREAD

**Discussion by JerryTomi** [Cyfrin/foundry-full-course-cu#2206] [General]
I have added my files to my contract folder but the zksync plugin is not responding and i couldn't deploy contract at least, is there any other way to go about this?

I have added the necessary interface and library to my contract

**Comment by EngrPips** [comment]
Hello @JerryTomi, Please show us what you are doing. I mean, show us your remix so we can be sure you are doing everything correctly.

**Reply by icodejsx** [reply]
![Screenshot (850)](https://github.com/user-attachments/assets/1572e252-9526-4cd9-8341-7003868e4b7d)

am having same issues here.. so please i cant see the zksync plugin

**Reply by EngrPips** [reply]
Hey @JerryTomi, seems to show up for me, Maybe you should another browser and ensure that your internet connection is stable

<img width="357" alt="Screenshot 2024-08-09 at 09 46 13" src="https://github.com/user-attachments/assets/0bf3305d-9c11-4b6d-9c84-4678fe7bf323">

**Reply by icodejsx** [reply]
what browser are you suing

**Reply by EngrPips** [reply]
I mean the browser on which you opened the remix website. You should also check that your internet connection is okay.

**Reply by JerryTomi** [reply]
Thank you

**Reply by EngrPips** [reply]
You are welcome, Friend.
